provider/aws/aws.go

```diff
@@ -120,7 +120,7 @@ func (p *AWSProvider) config() *aws.Config {
 		config.Endpoint = aws.String(p.Endpoint)
 	}
 
-	if os.Getenv("DEBUG") == "true" {
+	if os.Getenv("DEBUG") == "true" || os.Getenv("CONVOX_DEBUG") == "true" {
```

Comment:
I'm OK going with either or, but we should pick one. Having both can cause unexpected behavior while debugging.

Believe @ddollar made some changes in this regards a while back. Any thoughts?